src/Stnvh/Partial/Zip.php

```diff
@@ -64,9 +64,6 @@ private function receiveData($ch, $data) {
 	 * @return void
 	 */
 	public function __construct($url, $file = false) {
-		set_time_limit(15);
-		ob_start();
-
```

Comment:
This is deliberate; I use ```ob_start()``` in this instance to start capturing any unnecessary output, to then ```ob_flush()``` before setting headers to output the decompressed file contents. Saves any issues when setting headers after contents is output.